PixyViSy.cpp

```diff
@@ -1,91 +1,116 @@
 #include <PixyViSy.h>
 
-PixyViSy::PixyViSy(uint8_t sig, uint16_t calibration_dis_pix, uint8_t _goal_height)
+const uint16_t min_block_area = 100;
```

Comment:
I am also not sure if this is better than doing it via `#define` but I trust you have a good reason for that =).